checkstyle.xml

```diff
@@ -0,0 +1,446 @@
+<?xml version="1.0"?>
```

Comment:
Is this a duplicate of the file in `.github/workflows/checkstyle`? Is this required?